pom.xml

```diff
@@ -42,37 +41,14 @@
     </scm>
     <build>
         <plugins>
+         <plugin>
```

Comment:
No need to explicitly list these plugins as they are activated in the parent: https://github.com/apache/sling-parent/blob/7e6567a003f670e179895b9000e294bc631bae18/sling-bundle-parent/pom.xml#L192

Reply:
shame on me.. cargo culting - I removed the build block as it doesn't contain any further information